examples/sample_run_local_tools_groq_stream.py

```diff
@@ -53,7 +55,8 @@ def hello_tool(city: str):
     print(block)
     stream_storage.add(block)
 
+# Run the tools using the Toolhouse client with the created message
 messages += th.run_tools(stream_storage)
 
-response = client.chat.completions.create(model="gpt-4o", messages=messages, tools=th.get_tools() + local_tools)
+response = client.chat.completions.create(model="gpt-4o", messages=messages, tools=th.get_tools() + local_tools) # Retrieve tools installed from Toolhouse
```

Comment:
`gpt-4o` is an OpenAI model but the code uses Groq, and Groq does not serve OpenAI models. You should change this to any of the [Groq supported models](https://console.groq.com/docs/models) (suggest you use Llama 3.1 or 3.2)